kotlin/app/src/main/java/com/google/android/gms/samples/wallet/activity/CheckoutActivity.kt

```diff
@@ -59,7 +61,8 @@ class CheckoutActivity : AppCompatActivity() {
         setContentView(layout.root)
 
         // Setup buttons
-        googlePayButton = layout.googlePayButton.root
+        googlePayButton = layout.googlePayButton
+        googlePayButton.initialize(ButtonOptions.newBuilder().setAllowedPaymentMethods("VISA").build())
```

Comment:
Maybe we can use `PaymentsUtil.baseCardPaymentMethod()` here? Something like:

```
val paymentMethods: JSONArray = JSONArray().put(PaymentsUtil.baseCardPaymentMethod())

googlePayButton.initialize(ButtonOptions.newBuilder().setAllowedPaymentMethods(paymentMethods.toString()).build())
```
However this means to change the signature of the method to be non-private. WDYT?

Reply:
Good idea